runtimes/bun-1.0/helpers/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Fail build if any command fails
+set -e
+
+. /usr/local/server/helpers/before-build.sh
```

Comment:
what is this `.` supposed to be doing?

Reply:
it's the same as [`source`](https://linuxize.com/post/bash-source-command/) which is kind of like an import and execute the script in the current context.

Reply:
Yep, it's replacement for source. We use it in ~half of runtimes. Some (for some reason) simply don't have source. `.` is drop-in replacement, seems like. So far I didn't spot any issue with it

Reply:
For example in Java: https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/java-18.0/helpers/build.sh

Reply:
Alright got it 👍